static/js/formatters-internal.js

```diff
@@ -340,15 +324,28 @@ export function image(simpleOrComplexImage = {}, size = '200x', atLeastAsLarge =
         throw new Error("Invalid height specified");
       }
     }
-    const thumbnails = image.thumbnails
-      .filter(thumb => thumb.width && thumb.height)
-      .sort((a, b) => b.width - a.width);
-    return atLeastAsLarge
-      ? _getSmallestThumbnailOverThreshold(thumbnails, desiredWidth, desiredHeight)
-      : _getLargestThumbnailUnderThreshold(thumbnails, desiredWidth, desiredHeight);
+
+    const [urlWithoutExtension, extension] = _splitUrlOnIndex(image.url, image.url.lastIndexOf('.'));
+    const [urlBeforeDimensions, dimensions] = _splitUrlOnIndex(urlWithoutExtension, urlWithoutExtension.lastIndexOf('/') + 1);
+
+    if (desiredDims[0] === '' || desiredDims[1] === '') {
```

Comment:
I think you can combine/refactor this chunk of logic with the above lines 314-234 where it checks for `desiredDims[0|1] !== ''` so it's more concise

Reply:
updated!